tests/code/test_userManagement.py

```diff
@@ -67,21 +67,24 @@ def test_changeProgramInfo():
         eventName = "Test Event Name"
         contactName = "New Test Name"
         contactEmail = 'newtest@email'
+        location = "Danforth Tech"
         currentProgramInfo = Program.get_by_id(programId)
 
         assert currentProgramInfo.programName == "Adopt-a-Grandparent"
         assert currentProgramInfo.contactName == ""
         assert currentProgramInfo.contactEmail == ""
+        assert currentProgramInfo.contactEmail == ""
```

Comment:
Is this supposed to be .programLocation?

Reply:
fixed